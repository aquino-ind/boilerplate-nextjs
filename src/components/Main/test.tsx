import { render, fireEvent, screen } from '@testing-library/react';
import { Main } from '.';

describe('<Main />', () => {
    it('shoud render the heading', () => {
        const { container } = render(<Main />);
        expect(screen.getByRole('heading', { name: /meu texto asds top/i })).toBeInTheDocument();
        // expect(container.firstChild).toMatchSnapshot();
    });
});
