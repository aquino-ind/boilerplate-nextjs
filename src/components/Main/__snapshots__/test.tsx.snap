// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> shoud render the heading 1`] = `
.c0 {
  background-color: #5b00eb;
}

<div
  class="c0"
>
  <h1
    class="styles__Title-sc-1cs6c8-1"
  >
    meu texto top
  </h1>
</div>
`;
